lib/chat/views/chatroom/bloc/chat_action_bloc/chat_action_state.dart

```diff
@@ -0,0 +1,266 @@
+part of 'chat_action_bloc.dart';
+
+abstract class ChatActionState extends Equatable {}
+
+class ConversationInitial extends ChatActionState {
```

Comment:
why all the classes are added in the same file., plus if it required to keep in same file, add comments?

Reply:
they needs to be in the same file